data/pages-content/voting.md

```diff
@@ -4,23 +4,23 @@ title: Voting
 
 # Community Voting
 
-Welcome to EuroPython Community Voting! We want feedback about what YOU would like to see at EuroPython 2023. We have around 500 proposals covering many aspects of Python, and we are excited to be putting together an engaging, educational and entertaining programme.  Your feedback is an important ingredient in this refinement and curation process.
+Welcome to EuroPython Community Voting! We want feedback about what YOU would like to see at EuroPython 2023. We have around 550 proposals covering many aspects of Python, and we are excited to be putting together an engaging, educational and entertaining programme.  Your feedback is an important ingredient in this refinement and curation process.
 
 
 
 <div style={{textAlign: "center"}}>
 
-Community Voting will open between
+Community Voting is open between
 
-Monday, 27 March 14:00:00 CEST and
+Monday, 28 March 10:00:00 CEST and
 
-Sunday, 9 April 23:59:59 [Anywhere on Earth](https://en.wikipedia.org/wiki/Anywhere_on_Earth)
+Sunday, 02 April 23:59:59 [Anywhere on Earth](https://en.wikipedia.org/wiki/Anywhere_on_Earth)
 
-**Voting details will be published soon!**
+<ButtonLink href="https://voting.europython.eu">Take me to vote now!</ButtonLink>
```

Comment:
You're probably on it but this currently redirects to EP2022. So we need to ensure that this points to EP2023 when merged.

Reply:
This is correct,  the old app will be gone by then.